packages/radio-button/radio-button.scss

```diff
@@ -56,6 +59,39 @@ $radio-button-default-color--inverted: jkl.$color-svaberg;
     }
 }
 
+@mixin dot-error-background($dot-size, $gradient-step) {
+    /**
+     * Denne remsa med CSS trenger kanskje litt forklaring.
+     *
+     * tl;dr: er gjort sånn for å beholde avstanden mellom focus border og radioknappen med bakgrunnsfarge
+     *
+     * Bakgrunnen i ::before ødelegger for inset box-shadow på knappen. Skyggen
+     * som lager rammen rundt knappen blir liggende under bakgrunnsfargen. En
+     * metode for å løse det kan være å gjøre ::before-elementet mindre ved å gi
+     * top, right, bottom, left en positiv verdi. Ulempen med det er at focus-ringen
+     * da blir liggende helt inntill, noe som gjør den lite synlig. Ikke godt nok.
+     *
+     * Her utnytter vi derfor CSS som egentlig er laget for å bruke bilder som
+     * bakgrunn. Med et bilde kan man velge størrelsen og posisjonen bakgrunnen
+     * skal ta i elementet. Men her skal vi jo ha en _farge_, sier du?
+     * background-image behandler en gradient som et bilde. Så ved å lage en
+     * gradient utnytter vi dette smutthullet til å få tilgang på CSS for å styre
+     * størrelse og posisjon på fargebakgrunnen.
+     */
```

Comment:
Hvorfor ikke bare sette `background-color` på `.jkl-radio-button__dot`? `::before`-emelentet er vel spesifikt ment å være kun fokusringen.

Reply:
Jeg prøvde det på et tidspunkt, men med tastaturfokus forsvant fargen. Ser nå at `__dot::after` sin `background-color: transparent;` er det som påvirker `__dot` på den måten. Det oppdaget jeg ikke tidligere, så da ble det denne omveien 😅

Reply:
Det at fargen forsvinner kommer av at bakgrunnsfargen blir satt eksplisitt ved fokus, og dermed overstyrer feil-bakgrunnsfargen 😊 Siden vi uansett går mot dropp av IE kan man bare legge til
```scss
--checkbox-focus-background-color: #{$checkbox-error-background-color};
```
der hvor du nå har mixin-en for å fikse det